app/services/force/listing_service.rb

```diff
@@ -104,8 +99,9 @@ def self.array_sort!(listing)
       end
     end
 
-    private_class_method def self.get_listings(params = nil)
-      results = Request.new(parse_response: true).cached_get('/ListingDetails', params)
+    private_class_method def self.get_listings(params = nil, force = false)
```

Comment:
very open to feedback on how to handle passing "force" down to cache listings, I'm not super clear on what best practices are for ruby. also open to different naming of the var (e.g. force_recache/refresh instead of just force)

Reply:
@akegan what you have here is fine. let's get into other options in https://www.pivotaltracker.com/story/show/167411060, since that is more focused on changing the get listings path